nototools/noto_lint.py

```diff
@@ -855,13 +860,10 @@ def check_name_table():
             warn("name/license_url", "License",
                  "License URL doesn't match template: '%s'." % names[14])
 
-    def _check_needed_chars(cmap, char_filter):
-        # TODO(roozbeh): check the glyph requirements for controls specified at
-        # https://www.microsoft.com/typography/otspec/recom.htm
-
+    def _get_script_required(cmap):
         needed_chars = set()
         if font_props.script == "Qaae":  # Emoji
-            needed_chars = set()  # TODO: Check emoji coverage
+            needed_chars = _emoji_pua_set()  # legacy PUA for android emoji
```

Comment:
I would keep the existing TODO to actually check for the rest of emoji characters to be there too.